core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponentSpec.scala

```diff
@@ -481,24 +481,24 @@ class EntityComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers wit
    */
   it should "get entity subtrees from a list of entities" in withDefaultTestDatabase {
     withWorkspaceContext(testData.workspace) { context =>
-      val sampleSet1Paths = Seq(Seq(testData.sset1.toReference),
-                                Seq(testData.sset1.toReference, testData.sample1.toReference),
-                                Seq(testData.sset1.toReference, testData.sample1.toReference, testData.aliquot1.toReference),
-                                Seq(testData.sset1.toReference, testData.sample2.toReference),
-                                Seq(testData.sset1.toReference, testData.sample3.toReference),
-                                Seq(testData.sset1.toReference, testData.sample3.toReference, testData.sample1.toReference))
-      val sampleSet2Paths = Seq(Seq(testData.sset2.toReference),
-                                Seq(testData.sset2.toReference, testData.sample2.toReference))
-      val sampleSet3Paths = Seq(Seq(testData.sset3.toReference),
-                                Seq(testData.sset3.toReference, testData.sample5.toReference),
-                                Seq(testData.sset3.toReference, testData.sample6.toReference))
+      val sampleSet1Paths = Seq(EntityPath(Seq(testData.sset1.toReference)),
+                                EntityPath(Seq(testData.sset1.toReference, testData.sample1.toReference)),
+                                EntityPath(Seq(testData.sset1.toReference, testData.sample1.toReference, testData.aliquot1.toReference)),
+                                EntityPath(Seq(testData.sset1.toReference, testData.sample2.toReference)),
+                                EntityPath(Seq(testData.sset1.toReference, testData.sample3.toReference)),
+                                EntityPath(Seq(testData.sset1.toReference, testData.sample3.toReference, testData.sample1.toReference)))
```

Comment:
hm.  How does a Soft Conflict actually get used?  Maybe as long as you have one path to a soft-conflicting entity, you don't need all paths to it.